Formula/dotnet.rb

```diff
@@ -3,10 +3,11 @@ class Dotnet < Formula
   homepage "https://dotnet.microsoft.com/"
   # Source-build tag announced at https://github.com/dotnet/source-build/discussions
   url "https://github.com/dotnet/installer.git",
-      tag:      "v7.0.100-rtm.22521.12",
-      revision: "e12b7af219b96b5e07039ea8e3e268380329d72c"
-  version "7.0.100"
+      tag:      "v7.0.305",
+      revisoin: "98e1b6c38143f69c14ca5642ff28154a076abde6"
+  version "7.0.305"
```

Comment:
Source-build is only supported on 7.0.1xx.

This may change for .NET 8 but for .NET 7 it's documented [here](https://github.com/dotnet/source-build/#support).

Reply:
We could/should updated this to v7.0.108 however.